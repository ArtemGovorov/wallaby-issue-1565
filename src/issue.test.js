'use strict';

describe('wallaby-issue-1565', () => {
    it('example', () => {
        expect(__dirname).toMatchSnapshot();
    });
});
