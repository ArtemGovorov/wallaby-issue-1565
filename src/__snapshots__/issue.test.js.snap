// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`wallaby-issue-1565 example 1`] = `"<PROJECT_ROOT>/src"`;
